api/version.go

```diff
@@ -55,7 +55,7 @@ func VersionForType(nodeType NodeType) (Version, error) {
 // semver versions of the rpc api exposed
 var (
 	FullAPIVersion0 = newVer(1, 5, 0)
-	FullAPIVersion1 = newVer(2, 3, 0)
+	FullAPIVersion1 = newVer(2, 4, 0)
```

Comment:
The api version shouldn't be changed unless there are actual breaking changes which may break existing things which depend on the API.

Bumping this number means that everyone needs to also update lotus-miner / boost instances, which is really annoying (especially boost, which needs to ship a version supporting the new api)